.github/workflows/build-complete-samples.yml

```diff
@@ -902,6 +902,7 @@ jobs:
         run: |
           npm install
           npm run build --if-present
+          npm run test --if-present
```

Comment:
I'd like to run this action across the entire repo before merging, I think it might cause build breaks for certain samples. I don't seem to have the power to run this action against this branch though